app/api/custom/invoices.py

```diff
@@ -63,11 +65,9 @@ def order_invoices(order_identifier):
             or order.is_attendee(current_user)
         ):
             file_path = order.invoice_pdf_path
-            try:
-                return send_from_directory('../', file_path, as_attachment=True)
-            except FileNotFoundError:
+            if not os.path.exists(file_path):
```

Comment:
```suggestion
            if not os.path.isfile(file_path):
```

Reply:
@iamareebjamal - this change should be done everywhere then because i read that isfile will return true only when it points to a file while exist can return a directory also.

Reply:
pdf is a file, not a directory

Reply:
In my paste review, I wrote isfile only

Reply:
This is my bad apologies.